providers/default/defltprov.c

```diff
@@ -72,12 +99,20 @@ static const OSSL_ALGORITHM deflt_digests[] = {
     { "KECCAK_KMAC128", "default=yes", keccak_kmac_128_functions },
     { "KECCAK_KMAC256", "default=yes", keccak_kmac_256_functions },
 
-    { "SHAKE128", "default=yes", shake_128_functions },
-    { "SHAKE256", "default=yes", shake_256_functions },
+    /* Our primary name:NIST name */
+    { "SHAKE-128:SHAKE128", "default=yes", shake_128_functions },
+    { "SHAKE-256:SHAKE256", "default=yes", shake_256_functions },
 
 #ifndef OPENSSL_NO_BLAKE2
-    { "BLAKE2s256", "default=yes", blake2s256_functions },
-    { "BLAKE2b512", "default=yes", blake2b512_functions },
+    /*
+     * https://blake2.net/ doesn't specify size variants,
+     * but mentions that Bouncy Castle uses the names
+     * BLAKE2b-160, BLAKE2b-256, BLAKE2b-384, and BLAKE2b-512
+     * If we assume that "2b" and "2s" are versions, that pattern
+     * fits with ours.  We also add our historical names.
+     */
+    { "BLAKE2b-256:BLAKE2s256", "default=yes", blake2s256_functions },
```

Comment:
I assume that it is not the same as BLAKE2B-256 ??

Reply:
Oops